fireant/tests/slicer/widgets/test_highcharts.py

```diff
@@ -1638,43 +1638,43 @@ def test_invisible_y_axis(self):
 
 
 class HighChartsColumnChartTransformerTests(HighChartsBarChartTransformerTests):
-    chart_class = HighCharts.ColumnChart
+    chart_class = HighCharts.ColumnSeries
     chart_type = 'column'
 
 
 class HighChartsStackedBarChartTransformerTests(HighChartsBarChartTransformerTests):
     maxDiff = None
 
-    chart_class = HighCharts.StackedBarChart
+    chart_class = HighCharts.StackedBarSeries
     chart_type = 'bar'
     stacking = 'normal'
 
 
 class HighChartsStackedColumnChartTransformerTests(HighChartsBarChartTransformerTests):
-    chart_class = HighCharts.StackedColumnChart
+    chart_class = HighCharts.StackedColumnSeries
     chart_type = 'column'
     stacking = 'normal'
 
 
 class HighChartsAreaChartTransformerTests(HighChartsLineChartTransformerTests):
-    chart_class = HighCharts.AreaChart
+    chart_class = HighCharts.AreaSeries
     chart_type = 'area'
 
 
 class HighChartsAreaStackedChartTransformerTests(HighChartsAreaChartTransformerTests):
-    chart_class = HighCharts.AreaStackedChart
+    chart_class = HighCharts.AreaStackedSeries
     stacking = 'normal'
 
 
 class HighChartsAreaPercentChartTransformerTests(HighChartsAreaChartTransformerTests):
-    chart_class = HighCharts.AreaPercentageChart
+    chart_class = HighCharts.AreaPercentageSeries
     stacking = 'percent'
 
 
 class HighChartsPieChartTransformerTests(TestCase):
     maxDiff = None
 
-    chart_class = HighCharts.PieChart
+    chart_class = HighCharts.PieSeries
```

Comment:
Personal preference - but I think the name PieChart makes more sense to use than PieSeries.

Reply:
It's a series because there are multiple series within the same Chart. I changed the name to disambiguate it.